jina/serve/executors/__init__.py

```diff
@@ -250,10 +251,20 @@ async def __acall__(self, req_endpoint: str, **kwargs):
 
     async def __acall_endpoint__(self, req_endpoint, **kwargs):
         func = self.requests[req_endpoint]
-        if iscoroutinefunction(func):
-            return await func(self, **kwargs)
-        else:
-            return func(self, **kwargs)
+
+        _summary = (
+            self._summary_method.labels(
+                func.__name__, self.__class__.__name__, req_endpoint
```

Comment:
I am not sure, but maybe we should remove the method name as a label now that we always also expose endpoint? Its kind of redundant now? I would think that endpoint/executor tuple should be good enough?

Reply:
Yes makes sense

Reply:
You are right I will do